OmniKit/MessageTransport/MessageBlocks/StatusResponse.swift

```diff
@@ -9,42 +9,6 @@
 import Foundation
 
 public struct StatusResponse : MessageBlock {
-    
-    public enum DeliveryStatus: UInt8, CustomStringConvertible {
```

Comment:
Nice to see this duplication be cleaned up!

Reply:
Yup